open-metadata-implementation/access-services/digital-service/digital-service-api/README.md

```diff
@@ -0,0 +1,8 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# Digital Service Open Metadata Access Service (OMAS) API
+
+The Digital Service OMAS supports a REST API to support the lifecycle of an Egeria Digital Service
```

Comment:
Remove `Egeria`
API support the Java interface definition and common property objects.